src/utils/dateHelpers.ts

```diff
@@ -11,3 +11,9 @@ export const timestampToDateTimeString = (timestamp: number) =>
     hour12: false,
     minute: 'numeric',
   });
+
+export const timestampToString = (timestamp: number) =>
```

Comment:
I would rename it to `timestampToDateString` as you already have `timestampToDatetimeString` in the file and `timestampToString` is too ambigous.